### DISCUSSION TITLE
What is the purpose of [--adaptive:none] class?

### DISCUSSION THREAD

**Discussion by tppope** [htmlstreamofficial/preline#115] [Q&A]
Hi,

can someone explain me what is the purpose of the [--adaptive:none] class which is used in the dropdown menu on the line 65 and 77 in the following example? https://github.com/htmlstreamofficial/preline/blob/main/examples/html/sticky-floating-header.html

I can't found any documentation about it.

Thank you.

**Comment by melcarthus** [comment]
I was wondering the same thing. For anyone finding this and still looking for an answer:

I figured out what it does when I knew what settings the adaptive class takes.

Here is an example that helped me [https://preline.co/templates/agency/index.html](url)

Setting ```[--adaptive:adaptive]``` make the menu dropdown stick to the parent menu item. Setting the ```[--adaptive:none]``` removes this functionality and the dropdown takes its default position, most likely always position left.

[--adaptive:none]
[menu ---[link with dropdown]------------------- /menu]
[dropdown menu]

[--adaptive:adaptive]
[menu ---[link with dropdown]------------------- /menu]
----------[dropdown menu]
